platform/core/README.md

```diff
@@ -10,23 +10,7 @@ Logic shared across polyPod platforms - ios, android, wasm.
 ## Building
 
 On most systems, you can simply run `make`, and that will handle everything.
-
-### Building manually:
-
-If you can't use `make` for some reason, you can build the core (but not the
-Swift bindings), as follows:
-
-- Set up flatbuffers:
-
-        ./flatbuffers_shared/setup_flatbuffers.sh
-
-- Generate flatbuffers:
-
-        ./flatbuffers_shared/generate_flatbuffers.sh
-
-- Build the project:
-
-        cargo build
+Note: On Windows you will have to use a bash terminal, like GitBash.
```

Comment:
Maybe you should indicate that you can only build the Swift core in MacOS; at the end of the day, this is the only platform where you can actually run this meaningfully.
Anyway, add a link to GitBash, or how to obtain it. It seems to contain `curl` and `make`, so I guess we're good.
Maybe clarify that as long as you get those two commands independently, as well as git and an implementation of bash, we're good. Possibly the Linux subsystem for Windows will work too?

Reply:
This Makefile is also used to build the rust core, independently of building the Swift Core. There is also an early exist if `swift_xcframework` is built on other OS than MacOS.